### DISCUSSION TITLE
No funciona el botón visualizar que se encuentra en nota desde el detalle de la task

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#519] [4. Bug Reports] [answered]
Hola a todos

Adjunto un video en donde se muestra el inconveniente.

**Link:** https://drive.google.com/file/d/1keizmtbiMSQNbIsEhEEAYySkhEH1siPf/view?usp=sharing
Gracias =>

**Comment by acedward** [comment, marked as answer]
Creo que este bug fue corregido.

**Reply by NicolasPLG** [reply]
lo rev◙5saré muchas grac◙5as
